app/javascript/elm/src/Main.elm

```diff
@@ -922,6 +917,27 @@ viewSensorFilter sensors selectedSensorId =
         ]
 
 
+viewCrowdMapOptions : Bool -> Int -> Bool -> Html Msg
+viewCrowdMapOptions isCrowdMapOn crowdMapResolution isDisabled =
```

Comment:
You could pass the selectedSession in and checking here https://package.elm-lang.org/packages/krisajenkins/remotedata/latest/RemoteData#isSuccess